.storybook/preview-head.html

```diff
@@ -0,0 +1,6 @@
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+  rel="stylesheet"
+/>
```

Comment:
I'm thinking that this may only work within our storybook, but would not get bundled as part of the final package. I think we need to have it as a local asset.